swat/cas/table.py

```diff
@@ -3087,7 +3087,7 @@ def describe(self, percentiles=None, include=None, exclude=None, stats=None):
             percentiles = [25, 50, 75]
 
         if 50 not in percentiles:
-            percentiles = percentiles + [50]
+            percentiles.extend([50])
```

Comment:
I think in this case, it might be better just to append rather than creating a transient list.

`percentiles.append(50)`